test/test_location.js

```diff
@@ -432,7 +446,9 @@ describe('states.location', function() {
                 }
             });
```

Comment:
With `store_fields` gone, I'm wondering if this test is still necessary. If we do keep it, I think the test description needs to change.